test/tools/cli/IonProcess.java

```diff
@@ -392,15 +389,22 @@ private static void embeddedEventToIon(IonReader ionReader,
                         depth--;
                         tempWriter.stepOut();
                     } else if (event.getEventType() == EventType.SYMBOL_TABLE) {
-                        //TODO symbol table
+                        ionWriterForOutput.finish();
+                        ImportDescriptor[] imports = event.getImports();
+                        SymbolTable[] symbolTables = new SymbolTable[imports.length];
+                        for (int i = 0; i < imports.length; i++) {
+                            SymbolTable symbolTable = IonSystemBuilder.standard().build().newSharedSymbolTable(
+                                    imports[i].getImportName(), imports[i].getVersion(), null);
+                            symbolTables[i] = symbolTable;
+                        }
+                        ionWriterForOutput = args.getOutputFormat()
```

Comment:
Same comment about reassigning an IonWriter in the ProcessContext.

Reply:
Changed.